### ISSUE TITLE
Problem in list visualization

### ISSUE THREAD

**Issue by alexandrevl** [wekan/wekan#1530]
## Issue
**Impacted version**: 0.77

**Server Setup Information**: 

* Operating System: Ubuntu
* Deployment Method(tar/sandstorm/snap/docker/etc /http frontend (nginx, httpd, ...)): docker, nginx
* Node Version: 8.9.3
* MongoDB Version: NA
* ROOT_URL environment variable (Is there a subfolder?):

**Problem description**:
- Lists aren`t scrolling internally as expected.

<img width="1439" alt="screen shot 2018-03-08 at 13 57 53" src="https://user-images.githubusercontent.com/670957/37169354-77417f70-22e6-11e8-8591-f890cc997a74.png">

We are using a CSS Injector in Chrome to fix this problem, injecting this css: 

```css=
.board-wrapper .board-canvas .swimlane {
-webkit-box-align: start;
-moz-box-align: start;
-o-box-align: start;
-ms-flex-align: start;
-webkit-align-items: flex-start;
align-items: flex-start;
display: -webkit-box;
display: -moz-box;
display: -webkit-flex;
display: -ms-flexbox;
display: box;
display: flex;
-webkit-box-orient: horizontal;
-moz-box-orient: horizontal;
-o-box-orient: horizontal;
-webkit-flex-direction: row;
-ms-flex-direction: row;
flex-direction: row;
margin: 0 0 10px;
padding: 0 40px 5px 0;
overflow-x: auto;
overflow-y: hidden;
position: absolute;
left: 0;
right: 0;
top: 0;
bottom: 0
}
.list {
height: 100%;
}
```

**Comment by xet7** [comment]
@alexandrevl

Is this still a problem with latest Snap revision 145 or quay docker image latest tag?

**Comment by xet7** [comment]
@andresmanelli 

Can you check is this still needed, and include it if needed?

**Comment by andresmanelli** [comment]
I think this should be fixed in commits added after v0.77.
